### PR TITLE
Update Select stories to latest recommendations

### PR DESCRIPTION
**Pull request by jamie-lynch**
## What is the purpose of this change?

This change updates the Select stories to use the latest [recommendations](https://github.com/guardian/source/blob/main/docs/10-stories.md)

### CODE REVIEW

**Comment by jamie-lynch** [comment]
As part of this, I've removed the `constrainedWidth` css from the template so that it didn't appear in the code block. Any thoughts on whether it should have remained for the Playground and/or the Chromatic stories?

**Comment by SiAdcock** [comment]
> Any thoughts on whether it should have remained for the Playground and/or the Chromatic stories?

Let's try and keep custom CSS out of stories. It looks alright how you've implemented it

File: src/core/components/select/Select.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+				value={state}
+				onChange={(event) => setState(event.target.value)}
+			/>
+			<span css={message}>
+				{state ? `You have selected: ${state}` : ''}
+			</span>
+		</>
+	);
+};
+asChromaticStory(ControlledExample);
```
Comment (SiAdcock) [inline src/core/components/select/Select.stories.tsx:125]: We can probably get rid of this story, it's not really doing anything useful on Chromatic. If we need to demonstrate a code example of a controlled component, we can direct people to the [story in source-shakedown](https://github.com/guardian/source-shakedown/blob/main/stories/select/controlled.tsx)

**Review by SiAdcock — APPROVED** [review]
One small suggestion but this looks grand 🎹

**Pull request merged**